packages/expo-updates/ios/EXUpdates/UpdatesConfig.swift

```diff
@@ -11,6 +11,18 @@ public enum CheckAutomaticallyConfig: Int {
   case WifiOnly = 1
   case Never = 2
   case ErrorRecoveryOnly = 3
+  var asString: String {
+    switch self {
+    case .Always:
+      return "ON_LOAD"
```

Comment:
We should be deliberate about in which layer we do the translation. Where are the current translations done? (i.e. why isn't this `ALWAYS`). Should this be in the JS side of things? The reason I ask is that currently all the values in swift/kotlin match their enums, and this is making it different.

Reply:
Well right now, the translations happen in `config-plugins` (one way, from app.json to Expo.plist/AndroidManifest.xml). If desired, we can move this translation to the JS layer.

Reply:
Translation is moved to the JS layer.